contrib/options/label/Advanced

```diff
@@ -0,0 +1,29 @@
+#424242 Status: Needs feedback ; Feedback is needed
+#fbc02d Status: In progress ; Work is in progress
+#43a047 Status: Completed ; Work is complete
+#00796b Status: Help wanted ; 
+#880e4f Status: Blocked ; 
+#b0bec5 Status: Stale ; 
+
+#d32f2f Kind: Bug ; 
+#607d8b Kind: Question ; 
+#9c27b0 Kind: Security ; 
+#795548 Kind: Testing ; 
+#c62828 Kind: Breaking ; 
+#0288d1 Kind: Feature ; 
+#2e7d32 Kind: Enhancement ; 
+#f4511e Kind: Maintenance ; 
+#00796b Kind: Improvement ; 
+
+#546e7a Reviewed: Invalid ; Something was marked as invalid
+#616161 Reviewed: Duplicate; Something exists already
+#795548 Reviewed: Confirmed ; Something has been confirmed
+#eeeeee Reviewed: Wontfix ; Something won't be fixed
+
+#e53935 Legitimation: Needed ; A legitimation is needed
```

Comment:
I think these are way too specific use cases to be useful for most and imho should not be included

Reply:
Is this a case of LGTM being translated into another language and then back again? Otherwise I'm confused as to how a Legitimation would in general differ from Reviewed. Gitea has Approvals for PRs which should be shown on the pulls page.

Reply:
hmm I think we can remove LGTM labels since v1.12 has it buildIN; @opyale ping